.github/workflows/sync-fork.yml

```diff
@@ -0,0 +1,57 @@
+name: Sync Fork
+
+on:
+  schedule:
+    - cron: '0 0 */3 * *'  # runs at midnight every 3 days     
+
+jobs:
+  sync:
+    permissions: write-all
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: tgymnich/fork-sync@v1.8
+        with:
+          owner: SDSC-ORD
+          base: master
+          head: TopQuadrant:master
+          ignore_fail: true
+          retries: 1
+          retry_after: 15
+  fetch-release:
+    permissions: write-all
+    runs-on: ubuntu-latest
+
+    steps:
+     - name: Checkout repository
+       uses: actions/checkout@v3
+       with:
+          repository: 'TopQuadrant/shacl'
+          
+     - name: Get Version Tag
+       id: get_version_tag
+       run: |
+          git fetch --tags
+          git fetch --prune --unshallow || true
+          
+          LATEST_RELEASE=$(git describe --abbrev=0 --tags )
+          echo "latest-release=${LATEST_RELEASE}" >> $GITHUB_OUTPUT
+          echo "$LATEST_RELEASE"

```

Comment:
Minor thing, but this line is not needed anymore

Reply:
I removed line 39, I guess 37 and 38 are still necessary to be able to use the "latest release"  variable on line 40 and consequently 56